crawlerr/server.py

```
@@ -20,28 +25,49 @@
     sys.path.insert(0, str(Path(__file__).parent.parent))
     from crawlerr.config import settings
 
-# Set up logging configuration
+# Set up colorized logging configuration
 def setup_logging():
```

Ruff (0.12.2)

Missing return type annotation for public function `setup_logging`

Add return type annotation: `None`

(ANN201)